### PR TITLE
enhancement/1772 - Add airlines needed for ESSA

### LINKED ISSUE #1772: Add multiple airlines needed for ESSA
Add the following airlines needed for ESSA Stockholm/Arlanda (issue #1695 pull request #1696 )

- [x] Air Leap
- [x] Amapola Flyg
- [x] Nordica
- [x] Novair
- [x] West Air Sweden ~~(or its holding company West Atlantic, must find out which to use)~~

### PR DESCRIPTION
**Pull request by OlaRune**
Resolves #1772 

The purpose of this pull request is to add the needed airlines for ESSA Stockholm Arlanda.

### CODE REVIEW

File: assets/aircraft/js32.json
Comment (OlaRune) [inline assets/aircraft/js32.json:6]: Can't find `S` in [aircraft-format.md](https://github.com/openscope/openscope/blob/develop/documentation/aircraft-format.md), does it mean turboshaft? Maybe this should be added to the documentation?

File: assets/aircraft/js32.json
Comment (OlaRune) [inline assets/aircraft/js32.json:16]: For climb and descent rate, which "phase" from [eurocontrol](https://contentzone.eurocontrol.int/aircraftperformance/details.aspx?ICAO=JS32&ICAOFilter=js32) should I use? Right now I've used "initial climb" and "descent to FL100", but I don't know if that's correct.

File: assets/aircraft/js32.json
Comment (OlaRune) [inline assets/aircraft/js32.json:28]: The cruise speed at [eurocontrol](https://contentzone.eurocontrol.int/aircraftperformance/details.aspx?ICAO=JS32&ICAOFilter=js32) is in True Airspeed while the other are in Indicated Airspeed, is that okay?

File: assets/airlines/apf.json
Comment (OlaRune) [inline assets/airlines/apf.json:16]: I'm not sure if I should use the same F50 for the "Fokker 50 Freighter" model which is used for cargo flights.

File: assets/aircraft/atp.json
Comment (OlaRune) [inline assets/aircraft/atp.json:16]: As with the other aircraft in this PR I don't know which climb and descent values to use.

File: assets/aircraft/atp.json
Comment (OlaRune) [inline assets/aircraft/atp.json:18]: Yeah, I have no idea what to use for accelerate and decelerate. These are the values from the Jetstream 32 I did, which in turn are the same as the Jetstream 31.

File: assets/aircraft/atp.json
Comment (OlaRune) [inline assets/aircraft/atp.json:26]: Couldn't find, used the values from the Jetstream 32, which in turn uses the same as the Jetstream 31.

File: assets/aircraft/js32.json
Comment (OlaRune) [inline assets/aircraft/js32.json:18]: I used the same accelerate and decelerate as the Jetstream 31. Is that okay?

File: assets/aircraft/js32.json
Comment (OlaRune) [inline assets/aircraft/js32.json:26]: I only found a stall speed for the Jetstream 31 so I used that.

File: assets/airlines/est.json
Comment (OlaRune) [inline assets/airlines/est.json:12]: These are not their whole fleet, but the others are operated for LOT or SAS, so I only included the ones operated under their own name.

File: assets/airlines/swn.json
Comment (OlaRune) [inline assets/airlines/swn.json:12]: In more detail they use 767-200SF, is it okay to just use a 762 as an approximation?

Comment (OlaRune) [inline assets/airlines/swn.json:12]: Is it okay to use a 767-200 instead of an 767-200SF, which is the specific variant used?

File: assets/aircraft/atp.json
Comment (indianbhaji) [reply assets/aircraft/atp.json:18]: It's an estimate, I'd say 3 is fine :)

**Review by indianbhaji — CHANGES_REQUESTED** [review]
This is a preliminary review just looking at the comments you have made - I have resolved most of your comments, just one thing that needs changing :)

I've not crosschecked files/fleets as of yet, I'll sort this later tomorrow (today)

File: assets/aircraft/js32.json
Comment (OlaRune) [reply assets/aircraft/js32.json:6]: Okay, then I guess it should be `T` for both of my added aircraft? I'd also guess that the JS31 (not mine) needs changing as well?

Comment (OlaRune) [reply assets/aircraft/js32.json]: I have changed to `T` for both aircraft added in this PR.

File: assets/airlines/est.json
Comment (indianbhaji) [inline assets/airlines/est.json:3]: This airline is very confusing. It is now called Xfly apparently!
> In February 2020, Regional Jet announced it is rebranded as Xfly. Furthermore, an additional seven Embraer 190/195 will be leased to expand operations.
(see https://en.wikipedia.org/wiki/Nordica_(airline)

File: assets/airlines/est.json
Comment (indianbhaji) [reply assets/airlines/est.json:12]: This airline doesn't actually seem to fly any flights for themselves, appears to have entered into partnerships with SAS and LOT, and their flights appear to all be operated for LOT/SAS - even their booking site redirects you to LOT! I would be tempted to add all their current fleet as listed at https://www.planespotters.net/airline/Xfly.

File: assets/airlines/lpa.json
Comment (indianbhaji) [inline assets/airlines/lpa.json:12]: They only have one SB20

File: assets/airlines/swn.json
Comment (indianbhaji) [reply assets/airlines/swn.json:12]: West Air Sweden retired their 767s (https://www.scramble.nl/civil-news/west-air-sweden-retires-b767fs)

File: assets/airlines/swn.json
Comment (indianbhaji) [inline assets/airlines/swn.json:13]: 11 ATPs

File: assets/airlines/swn.json
Comment (indianbhaji) [inline assets/airlines/swn.json:14]: Missing 3 B738s

**Review by indianbhaji — CHANGES_REQUESTED** [review]
This was an interesting review to say the least...

A lot of research was required to find out more about the fleets for these airlines, hope this helps :)

File: assets/airlines/apf.json
Comment (OlaRune) [reply assets/airlines/apf.json:12]: Looking closer at their website it says that they have 14 Fokker 50 of which 3 are passenger aircraft. It also states that they have one Jetstream 32 (I have no idea why it says JS42 in my airline file...). I'll put those numbers in. Would it be okay if i had the fleets `default`, `passenger`, `cargo`? I wan't to have only passenger traffic for some lines at ESSA.

File: assets/airlines/est.json
Comment (OlaRune) [reply assets/airlines/est.json:3]: I agree that they are super confusing! I have changed their name to Xfly. Furthermore, I really don't think they actually went through with expanding their operations due to the pandemic, so I won't add their new shiny Embraer.

File: assets/airlines/est.json
Comment (OlaRune) [reply assets/airlines/est.json:12]: I updated their fleet to exactly what is presented at the linked page. (see other comment as well)

File: assets/airlines/lpa.json
Comment (OlaRune) [reply assets/airlines/lpa.json:12]: According to my original source they had 3 Saab 2000, I have no idea why it says eight there. Guess I was a bit tired and read it wrong, they kind of look the same...

However, I see that your source says that they have 1 active and 2 retired, so I guess my source was just a bit old (it said august 2020, but I guess the pandemic forced them to retire 2 since then). I've changed to 1.

File: assets/airlines/swn.json
Comment (OlaRune) [reply assets/airlines/swn.json:12]: Removed!

File: assets/airlines/swn.json
Comment (OlaRune) [reply assets/airlines/swn.json:13]: My numbers were a bit old,  updated!

File: assets/airlines/swn.json
Comment (OlaRune) [reply assets/airlines/swn.json:14]: Yup!

**Comment by OlaRune** [comment]
@indianbhaji Thanks for the very thorough review.

In general it seems I used too old data, but there were a few mistakes as well. Thanks for showing me that planespotters.net seems to have up to date numbers. Good to know for next time!

I answered all your comments but have not resolved them, and I have requested another review from @openscope/airport-reviewers,

File: assets/aircraft/js32.json
Comment (OlaRune) [inline assets/aircraft/js32.json:2]: @indianbhaji Apparently JS32 already existed, but with a capitalized filename. That has been changed in another pull request and therefore there is a conflict now. I see that some things differ between the files, for example the full aircraft name. Which one do you think is best? Mine or the old one?

Comment (OlaRune) [reply assets/aircraft/js32.json]: Alright, I've changed to using the one from develop.

**Comment by OlaRune** [comment]
@indianbhaji @erikquinn @Fechulo 

I have cleared all conflicts and all requested changes have been done. Would be great if I could have this PR approved and merged soon since it is blocking my other pull request! :)

**Review by Fechulo — APPROVED** [review]
All looks good to me, sorry for the wait ✔️

**Comment by OlaRune** [comment]
> All looks good to me, sorry for the wait ✔️ 

Thanks! No problem! ☺️

**Comment by OlaRune** [comment]
@erikquinn @Fechulo @indianbhaji 

Hi guys, I just wanted to point out that I still haven't abandoned my project of introducing Stockholm Arlanda to the sim. I just lost a bit of momentum for almost a year, hehe...

This pull request still blocks my work on the airport. Any chance it could be merged soon? @Fechulo has already approved it.

Happy new year!

**Comment by erikquinn** [comment]
Ah, didn't realize this was ready.

As a PSA (also for @Fechulo I suppose), when checking for PRs to merge, I filter by "approved review". If there is a "changes requested" review, that means it will not show up in the list of PRs that are approved, so I don't see it. So if a PR is ready, you'll have to dismiss any reviews that requested changes to show it should be merged. I'll do that now for this one 😃

**Pull request merged**
